spec/controllers/physical_rack_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe PhysicalRackController do
+  render_views
+
+  let!(:server) { EvmSpecHelper.local_miq_server(:zone => zone) }
+  let(:zone) { FactoryGirl.build(:zone) }
+
+  before(:each) do
+    stub_user(:features => :all)
+    EvmSpecHelper.create_guid_miq_server_zone
+    login_as FactoryGirl.create(:user)
+    ems = FactoryGirl.create(:ems_physical_infra)
+    @physical_rack = FactoryGirl.create(:physical_rack,
+                                        :ems_id => ems.id,
+                                        :id     => 1)
+  end
+
+  describe "#show_list" do
+    subject { get :show_list }
+
+    it do
```

Comment:
Please describe what are you testing here, i mean `it 'renders a GTL page'` or something like that.